app/database/migrations/2015_01_05_203341_CreateUsersTable.php

```diff
@@ -15,10 +15,11 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
-            $table->string('username', 200)->nullable(false);
-            $table->string('password')->nullable(false);
-            $table->rememberToken();
-            $table->string('email')->nullable()->default(null);
+            $table->string('username')->unique();
```

Comment:
That's fine if it's the case, so I assume it's the same with the rememberToken().